pkg/assembler/graphql/schema/certifyBad.graphql

```diff
@@ -54,6 +54,17 @@ PkgSrcArtObject is a union of Package, Source and Artifact. Any of these objects
 """
 union PkgSrcArtObject = Package | Source | Artifact
 
+# TODO: use this for ingestion for CertifyBad and HasSourceAt
+enum PkgMatchType {
+  ALL_VERSIONS
+  SPECIFIC_VERSION
+}
+
+input MatchFlags {
+  pkg: PkgMatchType!
+  depPkg: PkgMatchType!
+}
+
```

Comment:
These seem unused now. Will this be in the next PR?

Reply:
I can remove it but yes I was going to work on `CertifyBad` next.

Reply:
removed.

Reply:
made a note of it for the next PR all good :)